database/database.go

```diff
@@ -0,0 +1,151 @@
+package database
```

Comment:
I have been running this but it trips up on some of the formatting i.e. multiline comments